packages/components/src/Drawer/Drawer.component.js

```diff
@@ -124,11 +133,7 @@ DrawerContent.propTypes = {
 };
 
 function DrawerFooter({ children }) {
-	return (
-		<div className={theme['tc-drawer-footer']}>
-			{children}
-		</div>
-	);
+	return <div className={`tc-drawer-footer ${theme['tc-drawer-footer']}`}>{children}</div>;
```

Comment:
Could you change element by `footer` ? 
(I know it's not the purpose of the pull-request)